shownotes/S01E08-小雅：我在读研期间参与 CHAOSS 开源社区的心路经历.md

```diff
@@ -1,8 +1,8 @@
 ---
 num: 'S01E08'
 title: '[小雅：我在读研期间参与 CHAOSS 开源社区的心路经历](https://www.ximalaya.com/sound/513184857)'
-guests: '[小雅](https://github.com/xiaoya-Esther)'
-hosts: '[Wey Gu](https://github.com/wey-gu)、[majinghe](https://github.com/majinghe)、[Rick](https://github.com/linuxsuren)'
+guests: 'xiaoya-yaya'
+hosts: 'wey-gu)、majinghe、linuxsuren'
```

Comment:
How about this?
```diff
-hosts: 'wey-gu)、majinghe、linuxsuren'
+hosts: 'wey-gu, majinghe, linuxsuren'
```

or just remove the extra `)` after `wey-gu`

Reply:
yes, I missed that part.